stories/topics/proc-aws-uninstall-aap.adoc

```diff
@@ -1,12 +1,12 @@
 [id="ref-aws-uninstall-aap"]
 
-= Uninstalling {AAPonAWS}
+= Uninstalling {PlatformNameShort}
 
 .Procedure
 . In the AWS UI, navigate to the link:https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1[cloudformation page].
 Ensure you are in the correct region.
-. Select the stack for your {AAPonAWS} deployment.
+. Select the stack for your {PlatformNameShort} deployment.
 . Click btn:[Delete].
 . Click btn:[Delete] to confirm the deletion.
 
-It can take some time to complete deletion.
\ No newline at end of file
+It can take about 45 minutes to complete the deletion.
```

Comment:
how accurate is 'about 45 minutes'? Doesn't it depend on the system. What happens if it takes an hour. Do they panic?